ci/requirements-py27-min.yml

```diff
@@ -4,8 +4,8 @@ dependencies:
   - pytest
   - flake8
   - mock
-  - numpy==1.11
-  - pandas==0.18.0
+  - numpy==1.12
+  - pandas==0.20
```

Comment:
Anaconda doesn't have a pandas 0.19 build with numpy 1.12 so I've bumped another version to 0.20.